gtsam.h

```diff
@@ -1378,6 +1382,18 @@ virtual class Huber: gtsam::noiseModel::mEstimator::Base {
 
   // enabling serialization functionality
   void serializable() const;
+
+  double weight(double error) const;
+};
+
+virtual class Cauchy: gtsam::noiseModel::mEstimator::Base {
```

Comment:
You know, inheritance works in gtsam.h. Can you not just add weight to the Base?

Reply:
Limitation of wrap, understood.